extension/src/background/routing.ts

```diff
@@ -10,9 +9,7 @@ const frontend = new BrowserExtensionClient('frontend', new PortTransport())
 
 const studio = new BrowserExtensionClient(
   'studio',
-  STANDALONE_EXTENSION
```

Comment:
Removed this because running the extension standalone no longer works and this line of code affected my testing.